package.json

```diff
@@ -20,17 +20,17 @@
     "@google-cloud/functions-framework": "^1.7.1",
     "@google-cloud/storage": "^5.3.0",
     "cors": "^2.8.5",
-    "gtoken": "^5.0.3",
+    "gtoken": "^5.0.4",
     "json-mask": "^1.0.2",
-    "moment": "^2.29.0",
+    "moment": "^2.29.1",
     "superagent": "^6.1.0"
   },
   "devDependencies": {
-    "ava": "^3.12.1",
-    "eslint": "^7.9.0",
+    "ava": "^3.13.0",
+    "eslint": "^7.11.0",
     "minimist": "^1.2.5",
     "nyc": "^15.1.0",
-    "sinon": "^9.0.3",
-    "supertest": "^4.0.2"
+    "sinon": "^9.2.0",
+    "supertest": "^5.0.0"
```

Comment:
Not much in the way of changelog info for these, but they appear to be small changes and they're dev dependencies anyway.